revscoring/languages/persian.py

```diff
@@ -5,6 +5,12 @@
 from .language import Language, LanguageUtility
 
 DICTIONARY = enchant.Dict("fa")
+BADWORDS = set([
+    "کیرم", "ایتالیک", "کونی", "کیر", "فرمود", "آله", "فرموده", "فرمودند",
+    "جنده", "برووتو", "لعنت", "کون", "السلام", "جمهورمحترم", "کونی",
+    "کاکاسیاه", "آشغال", "گائیدم", "گوزیده", "مشنگ", "ننتو", "بخواب"
+])
+
```

Comment:
Shouldn't this be used to create a "is_badword" utility? See e.g.:
https://github.com/halfak/Revision-Scoring/blob/master/revscoring/languages/english.py#L51